server.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+
+const postsRouter = require("./posts/post-router");
+
+const server = express();
+
+server.use(express.json());
+
+server.use("/api/posts", postsRouter)
+
+module.exports = server
+
```

Comment:
I would strongly advise a "sanity check" here. It let's you check the functionality of your connection first.